examples/data_parallel/train_resnet_ddp.py

```diff
@@ -26,5 +27,7 @@ def _mp_fn(index):
 
 
 if __name__ == '__main__':
-  print('consider using train_resnet_spmd_data_parallel.py instead to get better performance')
-  xmp.spawn(_mp_fn, args=())
+  print(
+      'consider using train_resnet_spmd_data_parallel.py instead to get better performance'
+  )
+  torch_xla.launch(_mp_fn, nprocs=1)
```

Comment:
Why `nprocs=1`?

Reply:
I can leave it to nprocs=None as it is before.